src/e2e-test/features/BQExecuter/BQExecute_WithMacro.feature

```diff
@@ -0,0 +1,29 @@
+@BQExecute
+Feature: BigQueryExecute - Verify successful data transfer to BQ table using BQExecute plugin with macro arguments
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario:Validate successful records transfer Executes a SQL query in BigQuery Execute with macro arguments
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQueryExecute property sql "bqExecuteQuery"
+    Then Enter BigQueryExecute "projectId" as macro argument "bqeProjectId"
+    Then Enter BigQueryExecute "datasetProjectId" as macro argument "bqeDatasetProjectId"
+    Then Enter BigQueryExecute "serviceAccountFilePath" as macro argument "bqServiceAccount"
+    Then Enter BigQueryExecute "dataset" as macro argument "bqDataset"
+    Then Enter BigQueryExecute "table" as macro argument "bqTargetTable"
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the BigQueryExecute Properties
+    Then Save the pipeline
```

Comment:
Add preview step
and steps to capture preview logs

same comment applies to all scenarios

Reply:
@sawantpritam -done